lighthouse-core/audits/script-treemap-data.js

```diff
@@ -259,7 +234,31 @@ class ScriptTreemapDataAudit extends Audit {
         };
       }
 
-      nodes.push(node);
+      // If this is an inline script, place the node inside a top-level (aka depth-one) node.
+      // Also separate each iframe / the main page's inline scripts into their own top-level nodes.
+      const isInlineHtmlScript = script.startColumn || script.startLine;
+      if (isInlineHtmlScript) {
+        let htmlNode = htmlNodesByFrameId.get(script.executionContextAuxData.frameId);
+        if (!htmlNode) {
+          htmlNode = {
+            name,
+            resourceBytes: 0,
+            unusedBytes: undefined,
+            children: [],
+          };
+          htmlNodesByFrameId.set(script.executionContextAuxData.frameId, htmlNode);
+          nodes.push(htmlNode);
+        }
+        htmlNode.resourceBytes += node.resourceBytes;
+        if (node.unusedBytes) htmlNode.unusedBytes = (htmlNode.unusedBytes || 0) + node.unusedBytes;
+        node.name = script.content ?
+          '(inline) ' + script.content.trimStart().substring(0, 15) + '…' :
```

Comment:
Since we are applying the "(inline)" here, could we remove this code in the treemap viewer:
https://github.com/GoogleChrome/lighthouse/blob/0b0077d95a44cfa56e1d8eec3f0691f66d94ccbd/treemap/app/src/main.js#L87-L89

It would allow us to remove the document url dependency from the treemap viewer.